origin-dapp/src/components/listing-create.js

```diff
@@ -137,6 +145,9 @@ class ListingCreate extends Component {
         console.error(`Error fetching contract or IPFS info for listing: ${this.props.listingId}`)
         console.error(error)
       }
+    } else {
+      this.props.history.push('/')
```

Comment:
The problem might be in this line of code. When I have messaging enabled and try to create a new listing I just get redirected to home page without any message.

Reply:
Thanks 😬 I'll look into it later today.

Reply:
Good catch @sparrowDom 🤦‍♂️